src/main/java/dev/openfeature/sdk/FeatureProvider.java

```diff
@@ -1,5 +1,6 @@
 package dev.openfeature.sdk;
 
+import javax.annotation.Nullable;
```

Comment:
We don't allow usage of javax. We actually removed it from the SDK some time ago because it's very explicitly not allowed at some orgs, but it comes in transitively to this repo. Believe it or not, the `Nullable` annotation is present at runtime too.

Reply:
Out of curiosity are there other classes which should not be used, might be worth adding archUnit tests preventing those imports. But we should define a list for that.

Reply:
Dropped the @Nullable hint (even though it would've been a nice touch). 
Maybe we could consider creating our own annotation for that?

Reply:
> Dropped the @nullable hint (even though it would've been a nice touch). Maybe we could consider creating our own annotation for that?

Not a bad idea, I think. If we could keep it to compile time. But I don't think it would have as much utility as the javax ones since code quality tools wouldn't notice it - still might be helpful in IDE's though.